lib/Model/index.ts

```diff
@@ -937,7 +937,8 @@ export class Model<T extends DocumentCarrier = AnyDocument> {
 			getItemParams.ConsistentRead = settings.consistent;
 		}
 		if (settings.attributes) {
-			getItemParams.ProjectionExpression = settings.attributes.join(", ");
+			getItemParams.ProjectionExpression = settings.attributes.map((attribute) => "#" + attribute).join(", ");
+			getItemParams.ExpressionAttributeNames = settings.attributes.reduce((accumulator, currentValue) => (accumulator["#" + currentValue] = currentValue, accumulator), {});
```

Comment:
So ideally I'd like a better solution for this long term. But I think it's outside of the scope of this PR. Let's at least make it consistent with the codebase tho.

```suggestion
			getItemParams.ProjectionExpression = settings.attributes.map((attribute, index) => `#a${index}`).join(", ");
			getItemParams.ExpressionAttributeNames = settings.attributes.reduce((accumulator, currentValue, index) => (accumulator[`#a${index}`] = currentValue, accumulator), {});
```

Hopefully that code works 🤞

Reply:
It worked. Missed that codebase issue.